### PR TITLE
Fix migration 0088*

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
Migration 0088* was responsible for making `Event.slug` a required
field.  This resulted in a data migration: some events that didn't have
any slug needed one.

In testing everything worked fine, but in production 2 events ended up
with the same slug "none-".  I decided to fix the migration in place by
adding a random suffix to the event's slug.

### CODE REVIEW

File: workshops/migrations/0088_auto_20160530_1029.py [hunk 1]
```diff
@@ -2,20 +2,30 @@
...
+                with transaction.atomic():
+                    event.slug = slugify('{}-{}'.format(event.start,
+                                                        event.venue))
+                    event.save()
+            except IntegrityError:
+                population = string.ascii_letters + string.digits
+                random_suffix = ''.join(random.sample(population, 20))
+                event.slug = slugify('{}-{}-{}'.format(event.start,
+                                                       event.venue,
+                                                       random_suffix))
```
Comment (narayanaditya95) [inline workshops/migrations/0088_auto_20160530_1029.py]: We arent saving the event in the `except` block.

Comment (pbanaszkiewicz) [inline workshops/migrations/0088_auto_20160530_1029.py]: Thanks @narayanaditya95! Good catch!

**Comment by narayanaditya95** [comment]
+1 for merge.

**Comment by pbanaszkiewicz** [comment]
@narayanaditya95 I'm going to add some help_text to Event.slug in this PR yet. WIP.

**Comment by pbanaszkiewicz** [comment]
@narayanaditya95 re-review please. Thanks!

**Comment by narayanaditya95** [comment]
@pbanaszkiewicz 
Everything looks good. Ship it.

**Comment by pbanaszkiewicz** [comment]
Thanks @narayanaditya95

**Pull request merged**